### DISCUSSION TITLE
C language macro jump error

### DISCUSSION THREAD

**Discussion by xhwang-China** [microsoft/vscode-cpptools#9162] [Q&A]
# cpp tools not dump to what i want.
For example, the following code. 

```
#include <stdio.h>
#define FITST_DEFINE    0

#define SECONDARY_DEFINE (FITST_DEFINE+1)

void main(void)
{
    printf("%d\r\n", SECONDARY_DEFINE);
}
```

when I want to jump to define "SECONDARY_DEFINE" where the code in "printf("%d\r\n", SECONDARY_DEFINE);",  I want jump to "#define SECONDARY_DEFINE (FITST_DEFINE+1)",in fact,it will jump to  "#define FITST_DEFINE    0".
is this is feature? and How do I set up to achieve my goals.

thanks

**Comment by Colengms** [comment]
I suspect you are seeing a repro of this issue: https://github.com/microsoft/vscode-cpptools/issues/8577

However, I'm not able to repro the issue as simply as described (on Windows, with cl.exe set as the compiler).  If #8577 is not the same issue, could you provide more information about your configuration (platform, compiler, etc.), so we can try to reproduce this issue?

**Reply by xhwang-China** [reply]
oh，thanks，#8577 is the same issue,and i see it will be resolved in the next version.
looking forward to!
